src/Javis.jl

```diff
@@ -272,9 +274,22 @@ function render(
         tempdirectory = mktempdir()
     end
 
+    
+    frame_template = convert.(RGB, get_javis_frame(video, objects, 1; layers = layers))
+    frames_size = size(frame_template)
+    frames = postprocess_frames_flow(frames)
+
     filecounter = 1
     @showprogress 1 "Rendering frames..." for frame in frames
         frame_image = convert.(RGB, get_javis_frame(video, objects, frame; layers = layers))
+        if postprocess_frame != identity
+            frame_image = postprocess_frame(frame_image, frame, frames)
+            if any(size(frame_image) .> frames_size)
```

Comment:
I think it could be that one frame is larger in one dimension but smaller in the other so that first I reduce the one that is larger than enlarge the smaller one, that was the idea otherwise one can just check the passed function doesn't change the size.

Reply:
Oh I was reading that wrongly somehow I thought it was for all frames...